webknossos/examples/download_tiff_stack.py

```diff
@@ -15,7 +15,7 @@ def main() -> None:
     )
     mag_view = dataset.get_layer(LAYER_NAME).get_mag(MAG)
 
-    z = mag_view.bounding_box.topleft.z
+    z = mag_view.bounding_box.get_bounds("z")[0]
```

Comment:
Same here.

Reply:
I changed it to `.bounding_box.topleft_xyz.z`